edx-platform/pearson-pols-theme/lms/templates/ccx/course_name.html

```diff
@@ -0,0 +1,84 @@
+<%page expression_filter="h"/>
+<%!
+  from django.utils.translation import ugettext as _
+  from openedx.core.djangolib.js_utils import (
+    dump_js_escaped_json, js_escaped_string
+  )
+%>
+
+<style>
+  .ccx-change-course-name-feedback {
+    display: none;
+    margin: 0.5em 0;
+  }
+
+  .ccx-change-course-name-spinner {
+    display: none;
+    -webkit-animation: ccx-spinner-loading-rotate 2s infinite linear; /* Safari 4+ */
+    -moz-animation:    ccx-spinner-loading-rotate 2s infinite linear; /* Fx 5+ */
+    -o-animation:      ccx-spinner-loading-rotate 2s infinite linear; /* Opera 12+ */
+    animation:         ccx-spinner-loading-rotate 2s infinite linear; /* IE 10+, Fx 29+ */
+  }
+
+  @-webkit-keyframes ccx-spinner-loading-rotate {
+    0% {transform: rotateZ(0deg);}
+    100% {transform: rotateZ(360deg);}
+  }
+
+  @-moz-keyframes ccx-spinner-loading-rotate {
+    0% {transform: rotateZ(0deg);}
+    100% {transform: rotateZ(360deg);}
+  }
+
+  @-o-keyframes ccx-spinner-loading-rotate {
+    0% {transform: rotateZ(0deg);}
+    100% {transform: rotateZ(360deg);}
+  }
+
+  @keyframes ccx-spinner-loading-rotate {
+    0% {transform: rotateZ(0deg);}
+    100% {transform: rotateZ(360deg);}
+  }
+</style>
+
+<h2 class="hd hd-2">${_("Change CCX Course Name")}</h2>
+<hr>
+<p>${_("Please enter the new name for your CCX course.")}</p>
+
+%if ccx:
+  <form id="ccx-change-course-name-form" class="ccx-change-course-name-form" method="patch" data-remote="true" action="/api/ccx/v0/ccx/${ccx.locator}">
+    <input id="ccx-display-name" type="text" name="display_name" value="${ccx.display_name | n, js_escaped_string}" style="margin: 0.5em 0;"></input></br>
+    <input type="submit" name="enrollment-button" class="enrollment-button" value="${_('Change CCX course name.')}">
+    <i class="fa fa-refresh fa-2x ccx-change-course-name-spinner" aria-hidden="true"></i>
+  </form>
+  <div class="ccx-change-course-name-feedback"></div>
+% endif
+
+<script>
+  $ccxChangeCourseNameForm = $("#ccx-change-course-name-form");
+  $ccxChangeCourseNameSpinner = $(".ccx-change-course-name-spinner");
+  $ccxChangeCourseNameFeedBack = $(".ccx-change-course-name-feedback");
+
+  $ccxChangeCourseNameForm.submit(function() {
+    $("input[type='submit']", this).attr("disabled", "disabled");
+    $ccxChangeCourseNameSpinner.css("display", "inline-block");
+  });
+
+  $ccxChangeCourseNameForm.on("ajax:complete", function() {
+    $("input[type='submit']", this).removeAttr("disabled");
+    $ccxChangeCourseNameSpinner.css("display", "none");
+    $ccxChangeCourseNameFeedBack.css("display", "block");
+  });
+
+  $ccxChangeCourseNameForm.on("ajax:success", function(event, data, status, xhr) {
```

Comment:
is this line necessary ? if the ajax request is completed, will the spinner  be hidden by the line 69 ?

Reply:
I added a message on success. @andrey-canon Check again.